app/pubspec.yaml

```diff
@@ -74,7 +74,6 @@ dependencies:
   firebase_messaging: ^14.2.5
   firebase_performance: ^0.9.0+14
   firebase_storage: ^11.0.14
-  flare_flutter: ^3.0.2
```

Comment:
Don't remove it here. We need it both in `app/pubspec.yaml` and `lib/sharezone_widgets/pubspec.yaml`.